specification/maps/data-plane/Search/preview/2023-04-01-preview/search.json

```
@@ -658,7 +658,7 @@
                 "$ref": "#/parameters/Location"
               },
               {
-                  "name": "userCircularMapView",
+                  "name": "circularMapView",
                   "in": "query",
                   "description": "Optional. A circular geographic region specified by a comma separated list of the latitude and longitude of the center of the circle, and the radius of the circle, in meters.\n\nExample: 48.604311,-122.981998,5000\n\nNote. Mutually exclusive from userMapView.\n",
```

`userMapView` seems to be a typo? It should be `bbox`?
```suggestion
                  "description": "Optional. A circular geographic region specified by a comma separated list of the latitude and longitude of the center of the circle, and the radius of the circle, in meters.\n\nExample: 48.604311,-122.981998,5000\n\nNote. Mutually exclusive from `bbox`.\n",
```